app/vendor/ToolsAPI.js

```diff
@@ -279,7 +279,7 @@ sap.ui.define(['jquery.sap.global', 'sap/ui/core/library', 'sap/ui/Global', 'sap
                 if (bindingContextModel) {
                     model.mode = bindingContextModel.getDefaultBindingMode();
                     model.type = bindingContextModel.getMetadata().getName();
-                    model.data = bindingContextModel.getData ? bindingContextModel.getData() : undefined;
+                    model.data = bindingContextModel.getData ? bindingContextModel.getData('/') : undefined;
```

Comment:
this might make sense, but this will run into trouble with the i18n-models:
* using oData: resourceBundle will then return its config as it is this.oData there  
* using getData("/") will return nothing for i18n models
* using getProperty("/"); with that we would not have to use any internals and getProperty should be available on all models. i18n models would then return simply "/"

Unless we have a special treatment for i18n models I think getData("/") would at least give the least flawless result (and is anyways calling getProperty() under the hood for oData models.